utils/src/wizard/AzureWizardPromptStep.ts

```diff
@@ -15,7 +16,10 @@ export abstract class AzureWizardPromptStep<T extends types.IActionContext> impl
     public propertiesBeforePrompt!: string[];
     public prompted!: boolean;
     public id?: string;
-    public addedNumberOfActivityChildren?: number | undefined;
+
+    constructor(options?: { id?: string }) {
```

Comment:
I guess my comment about using a uuid all the time more applies to here, not on stepId.

Reply:
Yeah I don't mind making the id required and just assigning a uuid to it. Not sure where id was being used before and if assigning a uuid to it could break anything though.

Reply:
> I guess my comment about using a uuid all the time more applies to here, not on stepId.

Yeah agreed, I think it's probably okay / good to make the prompt step id required, but I wouldn't want to make the activity child require the step id because not all of them actually need it.